coil-compose-core/src/commonMain/kotlin/coil3/compose/CrossfadePainter.kt

```diff
@@ -99,12 +99,14 @@ class CrossfadePainter(
 
         val isStartSpecified = startSize.isSpecified
         val isEndSpecified = endSize.isSpecified
-        if (isStartSpecified && isEndSpecified) {
-            return Size(
-                width = maxOf(startSize.width, endSize.width),
-                height = maxOf(startSize.height, endSize.height),
-            )
```

Comment:
May I ask why you chose to use maxOf for startSize and endSize?

Reply:
It was to ensure that the placeholder isn't cut off. It's also to have consistency with `CrossfadeDrawable`.